### PR TITLE
feat(inputs.kafka_consumer): Option to set default fetch message bytes

### PR DESCRIPTION
**Pull request by AlbertasB**
### Required for all PRs:

- [X] Updated associated README.md.
- [ ] Wrote appropriate unit tests.
- [X] Pull request title or commits are in [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/#summary)

This allows setting number of message bytes to fetch from the broker in each request (default 1MB). Similar to the JVM's `fetch.message.max.bytes`. Might be useful in some cases. I needed it specifically to make telegraf consume messages faster from Kafka cluster.

### CODE REVIEW

**Review by sspaink — APPROVED** [review]

File: plugins/inputs/kafka_consumer/README.md [hunk 1]
```diff
@@ -103,6 +103,13 @@ and use the old zookeeper connection method.
   ## '2 * max_processing_time'.
   # max_processing_time = "100ms"
 
+  ## The default number of message bytes to fetch from the broker in each
+  ## request (default 1MB). This should be larger than the majority of
+  ## your messages, or else the consumer will spend a lot of time
+  ## negotiating sizes and not actually consuming. Similar to the JVM's
+  ## `fetch.message.max.bytes`.
+  # fetch_message_bytes = 1048576
```
Comment (srebhan) [inline plugins/inputs/kafka_consumer/README.md:111]: Can you please rename to `consumer_fetch_default` as this is what you ultimately use it as such.

File: plugins/inputs/kafka_consumer/kafka_consumer.go [hunk 2]
```diff
@@ -43,6 +43,7 @@ type KafkaConsumer struct {
 	BalanceStrategy        string          `toml:"balance_strategy"`
 	Topics                 []string        `toml:"topics"`
 	TopicTag               string          `toml:"topic_tag"`
+	FetchMessageBytes      int32           `toml:"fetch_message_bytes"`
```
Comment (srebhan) [inline plugins/inputs/kafka_consumer/kafka_consumer.go:46]: How about making this a [`config.Size`](https://github.com/influxdata/telegraf/blob/bd516ae587a27d040b457c593e3d5fa6aaf40269/config/types.go#L15) so you can specify human-readable sizes like `1MB` or `1MiB`?

**Review by srebhan — COMMENTED** [review]
Hey @AlbertasB, thanks for the PR! I have two small comments. Can you please take a look?

**Comment by AlbertasB** [comment]
Hey, thanks for the review and the comments. Somehow I did not get the message :(. I will try to make those changes ASAP. Thanks again :)

**Review by reimda — CHANGES_REQUESTED** [review]
It's looking great, almost ready to merge. It looks like there's an extra file in the PR that needs to be removed: oryxBuildBinary. Maybe a test file that sneaked in?

**Comment by AlbertasB** [comment]
Ooops no idea how that got there, nice catch! Removed :)

**Review by reimda — APPROVED** [review]

**Pull request merged**